<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SenderForm</class>
 <widget class="QWidget" name="SenderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>150</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	
	background-color: rgb(211, 211, 211);
	border-radius:4px;
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="sizeLabel">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>70</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0M / 0M</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>90</y>
     <width>341</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="speedLabel">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>70</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0 M/s</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
